sympy/functions/special/delta_functions.py

```diff
@@ -48,7 +51,7 @@ class DiracDelta(Function):
        Where ``x_i``-s are the roots of ``g``
     5) ``DiracDelta(-x) = DiracDelta(x)``
```

Comment:
These should also be rewritten as LaTeX.

1. $\frac{d}{d x} \theta(x) = \delta(x)$
2. $\int_{-\infty}^\infty \delta(x - a)f(x)\, dx = f(a)$ and $\int_{a-\epsilon}^{a+\epsilon} \delta(x - a)f(x)\, dx = f(a)$
3. $\delta(x) = 0$ for all $x \neq 0$
4. $\delta(g(x)) = \sum_i \frac{\delta(x - x_i)}{|g'(x_i)|}$ where $x_i$ are the roots of $g$
5. $\delta(-x) = \delta(x)$